.github/workflows/diff_report.yml

```diff
@@ -69,7 +70,11 @@ jobs:
         grep "^Diff Regression patch config:" text | cat > temp
         sed 's/Diff Regression patch config: //' temp > patch_config_link
         echo ::set-output name=patch_config_link::$(cat patch_config_link)
-        
+
+        grep "^Report label:" text | cat > temp
+        sed 's/Report label: //' temp > report_label
+        echo ::set-output name=report_label::$(cat report_label)
```

Comment:
could `$(cat report_label)` be double quoted?

Reply:
I dont think it is necessary, it works like this. 
Why do you think it needs to be double quoted?

Reply:
I probably overuse double quotes, but I try to use them as much as possible to avoid word splitting when using bash variables. Not a big deal.